server/package.json

```diff
@@ -5,21 +5,25 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon ./src/server.js"
+    "start": "nodemon ./src/server.js",
+    "queue": "nodemon ./src/queue.js",
+    "video-server": "nodemon ./src/video-server.js"
   },
```

Comment:
Can we use another script for running the `start`, `queue`, and `video-server` by one single command? Maybe we can use something like `concurrently` npm package like the one below?

`"dev": "concurrently \"npm start\" \"npm run queue\"  \"npm run video-server\""`